mapdata/WarcraftLegacies/UnitData/Core/nchr.json

```diff
@@ -72,15 +72,15 @@
     {
       "Id": 1919969397,
       "Type": 2,
-      "Value": 0.25
+      "Value": 2.5
```

Comment:
This is hit point regeneration, right?
Legacies units have remarkably consistent hit point regeneration rates so an outlier like this should be indicated somehow. I'd just change the Raider core tooltip to be something like "mobile siege unit **with high hit point regeneration.**" (Replacing the non-buld part with whatever it is now).

Reply:
Added in